sympy/series/tests/test_limits.py

```diff
@@ -540,3 +541,7 @@ def test_issue_14377():
 
 def test_issue_15984():
     assert limit((-x + log(exp(x) + 1))/x, x, oo, dir='-').doit() == 0
+
+
+def test_issue_13575():
+    assert isinstance(limit(acos(erfi(x)), x, 1), Add)
```

Comment:
This could have a better test. Maybe test the evalf() of the result? Also the issue refers to `erf(I).as_real_imag()` which could be tested.

Reply:
Right now `erf(I).as_real_imag()` gives `(0, -I*erf(I))`. I'm not sure if this is a satisfactory result- I could close this if the issue isn't actually resolved.

Reply:
I think that since erf(I) is imaginary then that's fine as in
```julia
In [9]: x = Symbol('x', imaginary=True)                                                                                           

In [10]: x.as_real_imag()                                                                                                         
Out[10]: (0, -ⅈ⋅x)
```